subsys/fs/fuse_fs_access.c

```diff
@@ -65,8 +65,14 @@ static void release_file_handle(size_t handle)
 static bool is_mount_point(const char *path)
 {
 	char dir_path[PATH_MAX];
+	size_t len;
 
-	sprintf(dir_path, "%s", path);
+	len = strlen(path);
+	if (len >=  sizeof(dir_path)) {
+		return false;
+	}
+
+	strcpy(dir_path, path);
```

Comment:
Question: why not use `memcpy` here? You already know the length of the `path`, by the call at line 70.

Reply:
nothing in particular, we can move it to a memcpy and set a string terminator  '\0'

Reply:
changed to `memcpy`